redis/db/redisbp/prometheus.go

```diff
@@ -131,4 +132,30 @@ func (e exporter) Collect(ch chan<- prometheus.Metric) {
 		float64(stats.IdleConns),
 		e.name,
 	)
+
+	// Baseplate spec
+	ch <- prometheus.MustNewConstMetric(
+		redisprom.ActiveConnectionsDesc,
+		prometheus.GaugeValue,
+		float64(stats.TotalConns-stats.IdleConns),
+		e.name,
+	)
+	ch <- prometheus.MustNewConstMetric(
+		redisprom.IdleConnectionsDesc,
+		prometheus.GaugeValue,
+		float64(stats.IdleConns),
+		e.name,
+	)
+	// ch <- prometheus.MustNewConstMetric(
+	// 	redisprom.PeakActiveConnectionsDesc,
+	// 	prometheus.GaugeValue,
+	// 	float64(xxx),
+	// 	e.name,
+	// )
```

Comment:
that one is missing at this point. It is a bit annoying to implement this I'll need to lock to prevent race conditions.

Reply:
let's remove the commented out code and add a comment of `// TODO: implement redisprom.PeakActiveConnectionsDesc`

Reply:
I implemented it :p